app/jobs/after_transition_tasks_for_rejected_return_job.rb

```diff
@@ -13,20 +13,24 @@ def perform(submission, transition)
         already_auto_resubmitted = submission.previously_transmitted_submission && submission.previously_transmitted_submission.efile_submission_transitions.where(to_state: :resubmitted
         ).any? { |transition| transition.metadata.dig("auto_resubmitted") }
         unless already_auto_resubmitted
-          submission.transition_to!(:resubmitted, {auto_resubmitted: true})
+          submission.transition_to!(:resubmitted, { auto_resubmitted: true })
         end
       end
-      # message_class = message_class_for_state(submission.current_state)
-      # if message_class
-      #   ClientMessagingService.send_system_message_to_all_opted_in_contact_methods(
-      #     client: submission.client,
-      #     message: message_class,
-      #     locale: submission.client.intake.locale
-      #   )
-      # end
-    end
 
-    EfileSubmissionStateMachine.send_mixpanel_event(submission, "state_file_efile_return_rejected")
+      message_class = message_class_for_state(submission.current_state)
+
+      if submission.is_for_federal_filing?
```

Comment:
same here, `is_for_state_filing?` ?

Reply:
@mpidcock oh oops saw Tim's approval so thought it was good to merge, but yes same case here